.gitignore

```diff
@@ -133,3 +133,6 @@ dmypy.json
 methods/transient/calval
 methods/transient/.ipynb_checkpoints/
 methods/transient/error.log
+methods/transient/NISAR_SES_Secular_Requirement_Validation_v2.ipynb
```

Comment:
We should not add this to the git ignore file, as it may affect other users or yourself in the future.